packages/main/src/Menu.ts

```diff
@@ -153,17 +154,25 @@ type MenuBeforeCloseEventDetail = { escPressed: boolean };
 			type: HTMLElement,
 		},
 	},
+	bubbles: true,
+	cancelable: true,
 })
 
 /**
- * Fired after the menu is opened. **This event does not bubble.**
+ * Fired after the menu is opened.
  * @public
  * @since 1.10.0
  */
 @event("open")
 
 /**
- * Fired before the menu is closed. This event can be cancelled, which will prevent the menu from closing. **This event does not bubble.**
+ * Fired when the menu is being closed.
+ * @private
+ */
+@event("close-menu")
```

Comment:
This event was bubbling before and now it is not.

Reply:
I've tested it and confirmed that we do need this event to bubble from the `ui5-menu-item` to the `ui5-menu` so on mobile device all menus are closed on pressing the `Escape` key.